pkg/api/handlers/libpod/pods.go

```diff
@@ -530,8 +530,12 @@ func PodStats(w http.ResponseWriter, r *http.Request) {
 	query := struct {
 		NamesOrIDs []string `schema:"namesOrIDs"`
 		All        bool     `schema:"all"`
+		Stream     bool     `schema:"stream"`
+		Delay      int      `schema:"delay"`
```

Comment:
nit of a nit, I prefer the variables in a struct to be alpha ordered.  If you have other changes please consider tweaking, at the very least flip Delay/Stream to match below.  My OCD overfloweth....